components/Layout/Page.tsx

```diff
@@ -101,19 +120,29 @@ const lightTheme = {
     tertiary: '#240090',
     quarternary: '#3500d3',
   },
+  mediaQueries: {
+    small: `@media screen and (max-width: 768px)`,
+    medium: `@media screen and (max-width: 1024px)`,
+    large: `@media screen and (max-width: 1200px)`,
+  },
 }
 
 const darkTheme = {
```

Comment:
Why isn't this theme typed with `DefaultTheme`?

ref: https://styled-components.com/docs/api#typescript